dubbo-cluster/src/main/java/com/alibaba/dubbo/rpc/cluster/loadbalance/ConsistentHashLoadBalance.java

```diff
@@ -36,6 +36,8 @@
  */
 public class ConsistentHashLoadBalance extends AbstractLoadBalance {
 
+    public static final String NAME = "consistenthash";

```

Comment:
Is this line you added is useless? @dchack

Reply:
it's the same as RandomLoadBalance's codes.@lovepoem

Reply:
IMO, this is ok.